examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,74 @@
+## Multi-Tenant logging using Fluent-bit, EKS , Amazon MSK and  Amazon OpenSearch
+
+In this example,we will showcase how to send your application logs from EKS's selective namespace(useful when you are using namesapce as tenant isolatio) and forward logs Amazon Managed Service for Kafka to store andto OpeneSearch for visualization. To achieve this, we will deploy Fluent Bit as a DaemonSet to tail /var/log/containers/*.log on the EKS cluster and use fluent-bit annotations to configure desired parser for pod's ina namespace enabled for logging via variable using terraform.This will create one topic for each namespace(enabled for logging) in Amazon MSK and then using MSK connector for OpenSearch we will send these logs to Opensearch such that each namespace(tenant) will have one Index.In the end there is a link to OpenSearch multi-tenancy configuration using RBAC from OpneSearch .
+
+There is terraform code in terraform directory which you will use to create an EKS cluster, MSK cluster, Kafka custom plugin,MSK Connector for OpenSearch  and OpenSearch domain in one VPC.
+
+Reference Architecure ![Architecture](Ref-Architecture.png?raw=true "Title")
+This solution can be enhanced to fan out logs from each namespace to multiple destination to duplicate or selectively send to other destination likes S3 etc. you can write customer kafka consumer for logs which might require further processing/filteration.
+
+#### Pre-requisites
+
+* . A S3 bucket for terraform backend
+* . An EC2 instance or Cloud9 instace with IAM permissions to create resources(VPC/S3 bucket/EKS Cluster/MSK Brokers/OpenSearch Domain).A Administrator access IAM role is recommended.
+* . Install kubectl (https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) and  kafka clients binaries to verify your KAFKA brokers (optional).
+
+#### Instructions
+
+* To get started, edit 0-proivder.tf to update backend S3 bucket , region and key prefix.
+
+* Follow terraform instructions from section below ,terraform will create EKS cluster/MSK and OpenSearch compnonents and install fluent-bit in 'logging' namespace and also create a 'example' namespace.
+
+* [Optional] Edit 3-variables.tf to create/edit more namespaces and enable logging on them.
+```
+default = [
+    {
+      "name" : "logging",
+      "enable_logs_to_es" = false,
+    },
+    {
+      "name" : "example",
+      "enable_logs_to_es" = true,
+```
+* Note. Terraform code will create VPC and all required components. But your OpenSearch dashboard will not be accessible over internet, so you might consider using a AWS client VPN ( or any connectivity method to allow you access to dashboard). you can also launch use a Microsoft windows instance in same VPC and access it via RDP and then access your OpenSearch dashboard 
```

Comment:
nit: I'd put this may be before the above, since it sounds like an intro for this step. And then move into the steps.